db/seeds.rb

```diff
@@ -144,7 +144,7 @@ def add_comments_to(commentable, contents)
     この作者の本はどれも面白い。
     わかりやすかったです。
   TEXT
-  Book.all.each do |book|
+  Book.all.find_each do |book|
```

Comment:
ここ、rubocopの怒られたんですかね？
修正するなら
```suggestion
  Book.find_each do |book|
```

の方がシンプルです

Reply:
おっしゃる通り、rubocopに指摘されたのですが、意味があまり分かっていない状態で直していました💦 
修正しました。

Reply:
こちら、今回はこのままでよいですが、本来であれば `bundle exec rubocop` とすべきでしたね。
おそらく `bundle exec rubocop` とただの `rubocop` では実行結果が微妙に違うと思います（修正前のコードに戻した場合）。
これはなぜでしょうか？